docs/src/docs/asciidoc/end-to-end-gradle-guide.adoc

```diff
@@ -199,14 +208,13 @@ By default, the _default.iprof_ file, if not specified otherwise, is generated a
 
 Step 3. **Build an optimized native image with profiles**.
 This step involves copying the generated profile to the conventional directory, ensuring it is automatically used during the build process.
-Place the _default.iprof_ file in the _src/pgo-profiles/main_ directory, and then run:
+Place the _default.iprof_ file in the _src/pgo-profiles/_ directory, and then run:
```

Comment:
I think that's a mistake, it should be `src/pgo-profiles/main`

Reply:
Definitely no. @Y-Shcheholskyy, we need to revert text changes in end-to-end-gradle-guide.adoc and end-to-end-maven-guide.adoc